dashboard/test/factories/factories.rb

```diff
@@ -303,6 +319,20 @@
       end
     end
 
+    trait :with_multi_auth do
+      after(:create) do |user|
+        user.primary_authentication_option = create(:authentication_option,
+          user: user,
+          email: user.email,
+          hashed_email: user.hashed_email,
+          credential_type: 'google_oauth',
```

Comment:
This was a typo in my original method, should be `google_oauth2`

Reply:
This makes me think we should add all of the providers as constants in `AuthenticationOption`. Doesn't need to be right now, but I think it would be helpful. I'm going to add that to our [backlog](https://docs.google.com/document/d/1HC8D64pnJd0LLNK2lDmfEbMZqV8Kwc3OCGWSXZuNHn8/edit?usp=sharing)